docs/operations_guide/Common.md

```diff
@@ -206,7 +206,13 @@ To see the exact error you can look at (depending on the configuration of the Ex
   "additionalData": {}
 }
 ```
-
+#### Avro serialization problem
+Common problem is when your AVRO data scheme not correspond with the one Nussknacker expects. You will find then `Unknown magic byte!` error indicating that data you produced to the topic are not serialized properly.
```

Comment:
I would write "Avro data encoding" or sth similar, as it's not about schema itself?